### ISSUE TITLE
Easy way to bulk delete all up related resources on AWS?

### ISSUE THREAD

**Issue by profilernz** [apex/up#712]
Is there an easy way to bulk delete all up related resources on AWS short of deleting all apps one by one? We've had about 10 devs freely testing up for the last couple of months, and there's about 100 random apps and related resources in this AWS account needing deleting now...
Thanks

**Comment by tj** [comment]
Not built-in at the moment. The easiest would probably be to use use `aws-cli` to just loop through the CloudFormation stacks and delete each that way, the only resources remaining would be the `<name>-function` role per-app and the Lambda function itself.

**Comment by profilernz** [comment]
OK, thanks.

**Comment by sanderhahn** [comment]
One workaround for removing an individual stack is: change the name of the app in the `up.json` and then do `up stack delete`. Although it seems that CloudWatch logs will persist.

**Comment by profilernz** [comment]
Indeed, I ended up doing it that way, took me about an hour. But it won't delete resources in a different region very well, unless you specify the region also (some devs had accidentally pushed some apps to the wrong region).
